lib/config/presets/internal/default.ts

```diff
@@ -112,6 +112,12 @@ export const presets: Record<string, Preset> = {
     dependencyDashboardApproval: true,
```

Comment:
Reminder for me to add a link to the final preset from the [`dependencyDashboardApproval` config option docs](https://docs.renovatebot.com/configuration-options/#dependencydashboardapproval).